hca2scea-backend/utils.py

```diff
@@ -5,10 +5,68 @@
 import glob
 import re
 
+import requests as rq
```

Comment:
we should add requests to requirements.txt

Reply:
yes, will do.